### DISCUSSION TITLE
Should i slow down?

### DISCUSSION THREAD

**Discussion by na6wink** [codewars/codewars.com#2928] [General]
Hello all, i'm new here. I'm new to web development in general. I recently started a coding camp in junction with FCC. I've passed html and css in both my paid camp and also FCC certificate and have been learning JS for the past 2 weeks.

I stumbled upon a youtube video that suggested this page for staying sharp and expanding knowledge in JS. I quickly discovered doing 7kyu is quite difficult and very time consuming if i even manage to get the correct solution. 

I'm currently up to learning about the DOM and Query in my codecamp, and i'm curious if anyone thinks i should slow down or stop entirely until i reached a certain kyu level? i'm just not sure how developed my JS should be to continue on, any info is greatly appreciated! thank you! :)

**Comment by HungryVovka** [comment]
Hello. You can simply sort katas by "most completed" or by specific tags. If something doesn't work, that's fine. Try to complete another task, and return to the original one after a deeper study of the language.

Have a nice day!

**Reply by na6wink** [reply]
Thank you for sharing some wisdom, it's great appreciated! i've since continued on in my course up to dom/query events, and i occasionally hop back into codewars to get some practice. I think continuing forward was the right move, i've grown to understand things a bit more since i have!
